cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/async/runnable/UpdateTopicConfigurationRunnable.java

```diff
@@ -48,39 +57,90 @@
  */
 public class UpdateTopicConfigurationRunnable extends OperationRunnable {
   private static final Logger LOG = LoggerFactory.getLogger(UpdateTopicConfigurationRunnable.class);
-  protected final TopicReplicationFactorChangeParameters _topicReplicationFactorChangeParameters;
   protected final String _uuid;
+  protected Map<Short, Pattern> _topicPatternByReplicationFactor;
```

Comment:
we can not make them final because in [one constructor](https://github.com/linkedin/cruise-control/blob/2a2163e9b8ec83db748ee960fa2387b3d5048923/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/async/runnable/UpdateTopicConfigurationRunnable.java#L79) we may not initialize them.

Reply:
Would it be possible to initialize them to `null` in the constructor otherwise?